pkgs/development/interpreters/io/default.nix

```diff
@@ -32,6 +41,10 @@ stdenv.mkDerivation {
     sed -ie \
           "s/add_subdirectory(addons)/#add_subdirectory(addons)/g" \
           CMakeLists.txt
+    # Bind Libs STATIC to avoid a segfault when relinking
+    sed -i 's/basekit SHARED/basekit STATIC/' libs/basekit/CMakeLists.txt
+    sed -i 's/garbagecollector SHARED/garbagecollector STATIC/' libs/garbagecollector/CMakeLists.txt
+    sed -i 's/coroutine SHARED/coroutine STATIC/' libs/coroutine/CMakeLists.txt
```

Comment:
To detect future regressions can you add a checkPhase that just runs the interpreter to test if it does not segfault. Just in case we see more libraries that need static linking.

Reply:
i added a checkInstallPhase, because checkPhase did not find a lib.
I also looked into the upstream test suite and decided not to include it, because some of the tests are failing independently of nix (compared to Ubuntu).

Reply:
> because some of the tests are failing independently of nix (compared to Ubuntu).

If they fail because of how nixos works we can disable those tests.